tequilapi/endpoints/proposals.go

```diff
@@ -119,6 +125,16 @@ func (pe *proposalsEndpoint) List(resp http.ResponseWriter, req *http.Request, _
 		return float32(f)
 	}()
 
+	natCompatibility := req.URL.Query().Get("nat_compatibility")
+	if natCompatibility == "auto" {
```

Comment:
Could we get a constant for this somewhere?

Reply:
Updated